src/components/InLineNotification.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { fireEvent, render } from '@testing-library/react-native'
+import * as React from 'react'
+import InLineNotification, { Severity } from 'src/components/InLineNotification'
+
+describe(InLineNotification, () => {
+  it('does not render CTA when onPress function is not provided', async () => {
+    const { findByText } = render(
```

Comment:
i think you should also add an assertion that the title and description can be found